resources/custom/lang/en_us.lang

```diff
@@ -93,11 +130,16 @@ material.graphite_water=Graphite Water
 material.phosphate_water=Phosphate Water
 material.precious_runoff=Precious Runoff
 material.radioactive_runoff=Radioactive Runoff
+material.bentonite_clay_solution=Bentonite Clay Solution
+material.calcium_carbonate_solution=Calcium Carbonate Solution
 
 material.silicon_solution=Silicon Solution
 material.novolacs=Novolacs Photopolymer
 material.copper_sulfate=Copper (II) Sulfate
 
+material.flue_gas=Flue Gas
+material.exhauststeam=Exhaust Steam
```

Comment:
You have a lang entry named `exhaust_steam` in addition to this one. Are both needed? Additionally, I believe NuclearCraft has its own exhaust steam, so you likely do not need to generate a GT material for it.